package.json

```diff
@@ -27,12 +27,12 @@
     "cors": "^2.8.5"
   },
   "peerDependencies": {
-    "bedrock": "^3.1.0",
-    "bedrock-account": "2.x - 3.x",
-    "bedrock-profile": "4.x - 5.x",
-    "bedrock-express": "^3.1.0",
+    "bedrock": "^3.1.1",
+    "bedrock-account": "3.x - 4.x",
```

Comment:
As with `bedrock-profile` this should not be listed as a peerDependency since it is not used directly here.